src/pe/header.rs

```diff
@@ -41,6 +41,7 @@ pub struct CoffHeader {
     pub characteristics: u16,
 }
 
+/// The size of `COFF_MAGIC` + `CoffHeader`.
 pub const SIZEOF_COFF_HEADER: usize = 24;
```

Comment:
Not sure if we should change this value to exclude COFF_MAGIC. It would be a silent breaking change.

Reply:
I'm ok breaking the world if we get windows object file writers :) this is why i wanted something like this in before 1.0, because there's unknown unknowns that might break a stable api, which would otherwise have been seen

Reply:
Added another commit with this breaking change.